static/_redirects

```diff
@@ -369,7 +369,7 @@
 #/docs/user-guide/kubectl/v1.9/*     https://v1-9.docs.kubernetes.io/docs/reference/generated/kubectl/kubectl-commands/:splat 301
 #/docs/user-guide/kubectl/v1.10/*     /docs/reference/generated/kubectl/kubectl-commands/:splat 301
 /docs/user-guide/kubectl-conventions/     /docs/reference/kubectl/conventions/
-/docs/user-guide/kubectl-cheatsheet/     /docs/reference/kubectl/cheatsheet/
+/docs/user-guide/kubectl-cheatsheet/     /docs/reference/kubectl/cheatsheet/ 307
```

Comment:
Hi @neha-viswanathan ,
Would you check the netlify documentation. I don't think `307` is supported.